runtime/lua/bugreport.lua

```diff
@@ -0,0 +1,70 @@
+-- Copied and modified gist.github.com/soulik/82e9d02a818ce12498d1
+local function getOS()
+	local raw_os_name = ""
+
+	-- LuaJIT shortcut
+	if jit and jit.os then
```

Comment:
You can also match on os_uname

Reply:
Well, that was certainly useful. Thanks for the help!